src/Tasks.UnitTests/XslTransformation_Tests.cs

```diff
@@ -864,7 +864,6 @@ public void CompiledDllWithTwoTypes()
             string doubleTypePath = Path.Combine(dir, "double.dll");
 
             CompileDoubleType(doubleTypePath);
-
```

Comment:
When the braces are used only as a code block - so no class, method body, conditional, loop, etc. - I actually think that an empty line can help. Is it possible that the rule is not working as intended?

Reply:
cc @sharwell

Reply:
I see what you mean, but I don't understand why anyone would add braces around code like this? Are we able to just remove them?

Reply:
There are some niche uses for code blocks. It's nice to be able to reuse variable names sometimes:

```C#
{
    int x = 42;
    // ...
}
{
    string x = "hello";
    // ...
}
```

And the scope applies to `using` also, so I can imagine favoring code like this:

```C#
{
    using var x = new Resource();
    using var y = new Resource();
    using var z = new Resource();
    // ...
}
```

over:

```C#
using (var x = new Resource())
{
    using (var y = new Resource())
    {
        using (var z = new Resource())
        {
          // ...
        }
    }
}
```

But I guess it's generally avoidable. In this case I would vote for just removing it.

Reply:
I have also seen code blocks used in ifdef scenarios:
```C#
#if CONDITIONAL_THINGY
  if (this || that)
#endif
  {
    Thingy1();
    Thingy2();
  }
```
But in this particular case I also vote for removing the code block.

Reply:
@rokonec in your example the braces don't have a blank line before hand. So it wouldn't trigger the rule.

This rule is about a blank line before open braces.